src/connection_string.ts

```diff
@@ -72,9 +73,8 @@ export async function resolveSRVRecord(options: MongoOptions): Promise<HostAddre
   // the SRV record is resolved before starting a second DNS query.
   const lookupAddress = options.srvHost;
   const txtResolutionPromise = dns.promises.resolveTxt(lookupAddress);
-  txtResolutionPromise.catch(() => {
-    /* rejections will be handled later */
-  });
+  // eslint-disable-next-line github/no-then
+  txtResolutionPromise.then(undefined, squashError);
```

Comment:
same here

Reply:
I can, but would we consider naming squashError something to communicate the same thing? or do you prefer the comment?

Reply:
`handleRejection`?

Reply:
I think these are special cases.  Unlike other `.catch(noop)`s, we're not squashing the errors - they'll still be handled when we await the promise later.  We just need catch handlers here in case there's a rejection before we handle the promise.  It's useful to communicate that we still care about the rejection, but we can't handle it yet.